torch/nn/modules/container.py

```diff
@@ -27,21 +27,50 @@ def __init__(self, **kwargs: Any) -> None:
 
 
 class Sequential(Module):
-    r"""A sequential container.
-    Modules will be added to it in the order they are passed in the constructor.
-    Alternatively, an ordered dict of modules can also be passed in.
+    r"""A list of ``Module``s that acts as a ``Module`` itself.
+    A ``Sequential`` is fundamentally a list of ``Module``s, each with a 
```

Comment:
There are some formatting errors here, which can be seen in the generated doc artifacts (FYI these can be found by clicking "Details" next to the `pytorch_python_doc_build` check and selecting the Artifacts tab): https://11343754-65600975-gh.circle-artifacts.com/0/docs/generated/torch.nn.Sequential.html

I think for "`Module`s" to properly render, there should be a space after the backticks and before the "s".